internal/actions/cmd/thirdpartycheck/main.go

```diff
@@ -41,7 +41,6 @@ var (
 		"golang.org/",
 
 		// Third party deps (allowed).
-		"go.opencensus.io",
```

Comment:
I see this as a required dep in some of the veneers, I don't think this can be removed yet until it is completely out of the tree.

Reply:
Yes, I see that too now, thanks.

Reply:
Restored, thanks.